app/views/schedules/show.html.erb

```diff
@@ -1,49 +1,60 @@
 <%= title "My Schedule - Rails World 2024" %>
 
 <div class="flex flex-col items-center h-full bg-gray-light">
-  <div class="flex flex-col items-center w-full h-full max-w-screen-md px-5 py-10">
-    <div class="flex w-full mb-4">
-      <h1 class="text-4xl italic font-black text-gray-800">My Schedule</h1>
-    </div>
+  <div class="flex flex-col items-center w-full h-full max-w-screen-md px-5">
+    <div class="sticky top-[79px] z-20 flex flex-col items-center w-full pt-10 bg-grey-50">
+      <div class="flex w-full mb-4">
+        <h1 class="text-4xl italic font-black text-gray-800">My Schedule</h1>
+      </div>
 
-    <%= render partial: 'sessions/filters', locals: { resource: :schedule } %>
+      <%= render partial: 'sessions/filters', locals: { resource: :schedule } %>
+    </div>
 
-    <div class="flex flex-col w-full h-full">
-      <% if @sessions.present? %>
-        <% @sessions.group_by(&:starts_at).each do |starts_at, sessions| %>
-          <div class="flex items-center">
-            <div class="text-grey-800 text-2xl font-bold pt-2 pb-4">
-              <a id="<%= session_anchor(sessions.first) %>">
-                <%= starts_at.strftime("%H:%M") %>
-              </a>
+    <div class="flex flex-col items-center w-full h-full">
+      <div class="flex flex-col w-full h-full mb-50">
+        <% if @sessions.present? %>
+          <% @sessions.group_by(&:starts_at).each do |starts_at, sessions| %>
+            <%= content_tag(
+                  :div,
+                  class: [
+                    "sticky z-10 flex items-center mb-5 bg-grey-50",
+                    "top-[273px]": session_filter_params.present?,
+                    "top-[241px]": session_filter_params.blank?
```

Comment:
top needs to change if we have active filters, otherwise the sticky hour hides below the filters